week01/238. Product of Array Except Self.h

```diff
@@ -0,0 +1,33 @@
+//
+// Created by FNU Shariful on 12/9/22.
+//
+
+#ifndef TJIP_LEETCODE_REPO_238_PRODUCT_OF_ARRAY_EXCEPT_SELF_H
+#define TJIP_LEETCODE_REPO_238_PRODUCT_OF_ARRAY_EXCEPT_SELF_H
+class Solution {
+public:
+    vector<int> productExceptSelf(vector<int>& nums){
+        int rightProduct[nums.size()],leftProduct[nums.size()],output[nums.size()];
```

Comment:
https://stackoverflow.com/questions/24136120/what-is-wrong-with-using-arrays-dynamically-allocated-in-c

Don't do that.